homepage/HomePage.py

```diff
@@ -0,0 +1,7 @@
+from flask import Flask, render_template

```

Comment:
I don't understand what this file is for? How is it used by the tiger app?

Reply:
My mistake, pushed the wrong folder :(